solarforecastarbiter/utils.py

```diff
@@ -268,10 +269,15 @@ def hijack_loggers(loggers, level=logging.INFO):
         logger = logging.getLogger(name)
         old_handlers[name] = logger.handlers
         logger.handlers = [handler]
+        sentry_logging.ignore_logger(name)
     yield handler
     for name in loggers:
         logger = logging.getLogger(name)
         logger.handlers = old_handlers[name]
+        try:
+            sentry_logging._IGNORED_LOGGERS.remove(name)
```

Comment:
use of private attribute is worrisome, but I don't really understand what's happening here. I assume you considered an alternative so ok to move ahead.

Reply:
yeah there isn't a good alternative considering that this code is used by the api and doesn't access the sentry init where other tricks could be used. At least there is a test that will fail if this changes